server/auth/types/authentication_type.ts

```diff
@@ -26,6 +26,7 @@ import {
   IOpenSearchDashboardsResponse,
   AuthResult,
 } from 'opensearch-dashboards/server';
+import { any } from 'joi';
```

Comment:
Is this the correct import for any? Shouldn't this be the typescript primitive

Reply:
This import is not needed. removed. Thanks for pointing it out!